src/components/List/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { intersection, pick } from 'lodash'
```

Comment:
I highly suggest importing the methods from their subpackages or subpaths or what is that approach called. It results in significantly smaller bundle because only the functions that you actually use are bundled, thanks to Webpack's tree-shaking
https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/
```
library does not bring along a full version of lodash.
Tree-shaking is syntax-sensitive. To enable tree-shaking in lodash, you have to import functions using syntax like import foo from 'lodash/foo'. import { foo } from 'lodash' will not tree-shake, nor will, obviously, import _ from 'lodash'. Support for this syntax was implemented in Lodash v4.
```
Also, the per-method packages are, apparently, deprecated, as well as cumbersome https://lodash.com/per-method-packages

Reply:
Oh, I thought it was tree-shakeable when imported like that. Fixed 😊

Reply:
Sorry for confusion. Now it should be really fixed and ready for a new review.

Reply:
Yep, that should work 👍 Although, I've just noticed Rollup is used here, so not sure how that will play together.. I guess we'll have to experiment and see how the bundle size of whatever application is using this will change..